MonoGame.Framework/Game.cs

```diff
@@ -19,6 +19,10 @@ namespace Microsoft.Xna.Framework
 {
     public class Game : IDisposable
     {
+#if DESKTOPGL
+        public static bool Resizable = false;

```

Comment:
> was more sensible for the moment.

Maybe for the moment... but we definitely don't want to leave window creation in `GraphicsDevice`.

> it's only Resizable property that is not setable after creation.

We can maybe make due without that for now, but we should talk to the SDL team about it.  If OpenTK supported it I don't see why SDL couldn't.